### DISCUSSION TITLE
I compiled the archive  dasniko.keycloak-requiredaction.jar, but where to put it in key cloack?

### DISCUSSION THREAD

**Discussion by des1roer** [dasniko/keycloak-extensions-demo#5] [Q&A]
![image](https://user-images.githubusercontent.com/10970793/205621774-491f1af9-c000-49a5-a569-b49d0b53199e.png)

**Comment by bickelj** [comment]
From https://www.keycloak.org/docs/21.1.1/server_development/index.html#registering-provider-implementations
> Providers are registered with the server by simply copying them to the providers directory.
